source/client/renderer/GameRenderer.cpp

```diff
@@ -204,7 +204,9 @@ void GameRenderer::setupGuiScreen()
 {
 	float x = Gui::InvGuiScale * Minecraft::width;
 	float y = Gui::InvGuiScale * Minecraft::height;
+#ifndef __ANDROID__
 	glClear(GL_ACCUM);
```

Comment:
Was a thing on Android. Why remove this?

Reply:
it didn't really made much sense since GL_ACCUM does not exist in embedded opengl so i blocked it out. TheBrokenRail mentioned it uses GL_DEPTH_BUFFER_BIT